task/buildah-rhtap/0.1/buildah-rhtap.yaml

```diff
@@ -7,6 +7,10 @@ metadata:
   annotations:
     tekton.dev/pipelines.minVersion: "0.12.1"
     tekton.dev/tags: "containers, rhtap"
+    task.results.format: application/text
+    task.results.type: external-link
```

Comment:
Since we don't have an external sbom viewer mechanism, we can skip this annotation so that UI will open the sbom task logs modal to show the output instead of redirecting to external page. Once we have an external sbom viewer then we can add this annotation back.

example sbom logs modal:

![image](https://github.com/redhat-appstudio/build-definitions/assets/9964343/5ed5de5e-c976-4f94-9646-6dbb47260a53)



```suggestion
 
```

Reply:
So what does the `LINK_TO_SBOM` do? Does RHDH just automatically link to the logs of the task that emits a `LINK_TO_SBOM` result (when the `results.type` is not `external-link`)?

Does that mean the LINK_TO_SBOM result needs to be added to the task that prints the SBOM and the value of the result doesn't matter?

Reply:
But the build task ([buildah-rhtap](https://github.com/redhat-appstudio/build-definitions/blob/main/task/buildah-rhtap/0.1/buildah-rhtap.yaml) in this case) does not print the SBOM in its logs, so if I understood it correctly, RHDH would open am modal and show the logs, but the SBOM wouldn't be printed there, just the usual build logs.

Reply:
Right, [show-sbom](https://github.com/redhat-appstudio/build-definitions/blob/main/task/show-sbom/0.1/show-sbom.yaml) task needs to be annotated.

Reply:
@karthikjeeyar I completely reworked the PR. See if the new `show-sbom-rhdh` task has everything we need.